JasmineTests/Common/Models/TextGridTests.swift

```diff
@@ -0,0 +1,83 @@
+import XCTest
+@testable import Jasmine
+
+class TextGridTests: XCTestCase {
+    func testInitFromInitialGrid() {
+        let initialGrid = [["a", "b"], ["c", "d"]]
```

Comment:
should test larger dataset, and test dataset (matrix) with missing entries here and there.